src/PscIde/Command.purs

```diff
@@ -248,7 +263,8 @@ instance decodeTypeInfo :: DecodeJson TypeInfo where
     definedAt <- o `getFieldMaybe` "definedAt"
     expandedType <- o `getFieldMaybe` "expandedType"
     documentation <- o `getFieldMaybe` "documentation"
-    pure (TypeInfo { identifier, type', module', definedAt, expandedType, documentation })
+    exportedFrom <- Right $ either (const []) id $ getField o "exportedFrom"
```

Comment:
Do we need to reuse the `fallback` parser again here? I don't think 0.11.6 is going to happen anytime soon.